prescient/simulator/tests/test_simulator.py

```diff
@@ -14,28 +14,24 @@
 import pandas as pd
 import numpy as np
 
-from prescient.downloaders import rts_gmlc
 from prescient.scripts import runner
-from tests.simulator_tests import simulator_diff
-
 from prescient.simulator import Prescient
 
 this_file_path = os.path.dirname(os.path.realpath(__file__))
```

Comment:
You might consider using `pyomo.common.fileutils.this_file_dir()` here

Reply:
In this particular case, nah.